src/utils.ts

```diff
@@ -0,0 +1,118 @@
+   ${baseProperty('paddingX', 'padding-right')};
+   ${baseProperty('paddingY', 'padding-top')};
+   ${baseProperty('paddingY', 'padding-bottom')};
+   ${baseProperty('paddingTop', 'padding-top')};
+   ${baseProperty('paddingBottom', 'padding-bottom')};
+   ${baseProperty('paddingLeft', 'padding-left')};
+   ${baseProperty('paddingRight', 'padding-right')};
+   ${baseProperty('border', 'border')};
+   ${baseProperty('borderX', 'border-left')};
+   ${baseProperty('borderX', 'border-right')};
+   ${baseProperty('borderY', 'border-top')};
+   ${baseProperty('borderY', 'border-bottom')};
+   ${baseProperty('borderTop', 'border-top')};
+   ${baseProperty('borderBottom', 'border-bottom')};
+   ${baseProperty('borderLeft', 'border-left')};
+   ${baseProperty('borderRight', 'border-right')};
+   ${createResponsiveStyles(
+      (key) => css`
+         ${responsiveProperty(key, 'margin', 'margin')};
+         ${responsiveProperty(key, 'marginX', 'margin-left')};
+         ${responsiveProperty(key, 'marginX', 'margin-right')};
+         ${responsiveProperty(key, 'marginY', 'margin-top')};
+         ${responsiveProperty(key, 'marginY', 'margin-bottom')};
+         ${responsiveProperty(key, 'marginTop', 'margin-top')};
+         ${responsiveProperty(key, 'marginBottom', 'margin-bottom')};
+         ${responsiveProperty(key, 'marginLeft', 'margin-left')};
+         ${responsiveProperty(key, 'marginRight', 'margin-right')};
+         ${responsiveProperty(key, 'padding', 'padding')};
+         ${responsiveProperty(key, 'paddingX', 'padding-left')};
+         ${responsiveProperty(key, 'paddingX', 'padding-right')};
+         ${responsiveProperty(key, 'paddingY', 'padding-top')};
+         ${responsiveProperty(key, 'paddingY', 'padding-bottom')};
+         ${responsiveProperty(key, 'paddingTop', 'padding-top')};
+         ${responsiveProperty(key, 'paddingBottom', 'padding-bottom')};
+         ${responsiveProperty(key, 'paddingLeft', 'padding-left')};
+         ${responsiveProperty(key, 'paddingRight', 'padding-right')};
+         ${responsiveProperty(key, 'border', 'border')};
+         ${responsiveProperty(key, 'borderX', 'border-left')};
+         ${responsiveProperty(key, 'borderX', 'border-right')};
+         ${responsiveProperty(key, 'borderY', 'border-top')};
+         ${responsiveProperty(key, 'borderY', 'border-bottom')};
+         ${responsiveProperty(key, 'borderTop', 'border-top')};
+         ${responsiveProperty(key, 'borderBottom', 'border-bottom')};
+         ${responsiveProperty(key, 'borderLeft', 'border-left')};
+         ${responsiveProperty(key, 'borderRight', 'border-right')};
+      `
+   )}
+`;
+
+/**
+ * Get the normalized event key across all browsers
+ * @param event keyboard event
+ */
+export function normalizeEventKey(event: React.KeyboardEvent) {
+   const { key, keyCode } = event;
+
+   const isArrowKey = keyCode >= 37 && keyCode <= 40 && key.indexOf('Arrow') !== 0;
+
+   const eventKey = isArrowKey ? `Arrow${key}` : key;
```

Comment:
Doesn't `key` already contain "Arrow" if it's an arrow key?

Reply:
I think this is adding it only when it isn't prefixed with Arrow to account for browser differences?